venter/views.py

```diff
@@ -5,6 +5,7 @@
 from rest_framework.generics import get_object_or_404
 from rest_framework import viewsets
 from rest_framework.response import Response
+from nltk.corpus import stopwords
```

Comment:
You're using nltk here without downloading the required resources. You can either download `stopwords` or use the other PyPI package instead.